src/earn/EarnCollectScreen.test.tsx

```diff
@@ -384,6 +384,16 @@ describe('EarnCollectScreen', () => {
       </Provider>
     )
 
+    await waitFor(() => {
+      expect(queryByTestId('EarnCollect/RewardsLoading')).toBeFalsy()
+    })
+    await waitFor(() => {
+      expect(queryByTestId('EarnCollect/ApyLoading')).toBeFalsy()
+    })
+    await waitFor(() => {
+      expect(queryByTestId('EarnCollect/GasLoading')).toBeFalsy()
+    })
+
```

Comment:
this test was accidentally passing because the button was disabled from loading other info 😐